src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +21,26 @@ type HacbsApplicationDetailsProps = {
 const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicationName }) => {
   const { namespace } = React.useContext(NamespaceContext);
   const navigate = useNavigate();
+  const { quickStarts } = useChrome();
+
   const [application, applicationsLoaded] = useK8sWatchResource<ApplicationKind>({
     groupVersionKind: ApplicationGroupVersionKind,
     name: applicationName,
     namespace,
   });
   const appDisplayName = application?.spec?.displayName || '';
 
+  useEffect(() => {
+    const hacbsShowAppQs = localStorage.getItem('hacbs\\showApplicationQuickstart');
+    if (hacbsShowAppQs === null) {
+      localStorage.setItem('hacbs\\showApplicationQuickstart', 'false');
```

Comment:
nit: shall we use the file path separator `/` ? it is slightly more readable :) . wdyt?
 
```suggestion
      localStorage.setItem('hacbs/showApplicationQuickstart', 'false');
```

Reply:
thanks @karthikjeeyar... good suggestion, made this change in the latest push